### PR TITLE
 Add a runnable example of ocagent exporter.

### PR DESCRIPTION
**Pull request by songy23**
Mostly from https://opencensus.io/exporters/supported-exporters/go/ocagent/, with adding zPages functionality.

### CODE REVIEW

File: example/main.go [hunk 1]
```diff
@@ -0,0 +1,117 @@
+package main
```
Comment (bogdandrutu) [inline example/main.go]: Why not in cmd?

Comment (songy23) [reply example/main.go]: Updated, I'm good with either ways. Previously put under example/ since I think cmd/ may be used for other main launchers.

Comment (bogdandrutu) [reply example/main.go]: I see, I think it makes sense to keep it in the example then :), sorry.

Comment (songy23) [reply example/main.go]: Reverted, no worries.

File: example/main.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+	"contrib.go.opencensus.io/exporter/ocagent"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/trace"
+	"go.opencensus.io/zpages"
+)
+
+func main() {
+	zPagesMux := http.NewServeMux()
```
Comment (odeke-em) [inline example/main.go]: Remember the comment suggestion from the website PR? Please apply that here too.

Comment (songy23) [reply example/main.go]: SG, added comments here.

**Pull request merged**